src/components/Layout/TopNav/TopNav.tsx

```diff
@@ -300,7 +300,7 @@ export default function TopNav({
               <div className="flex items-center -space-x-2.5 xs:space-x-0 ">
                 <div className="flex md:hidden">
                   <button
-                    aria-label="Search"
+                    aria-label="Telusuri"
```

Comment:
Seems shorter and more familiar?

```suggestion
                    aria-label="Cari"
```

![Screenshot_20230627-151932.png](https://github.com/reactjs/id.react.dev/assets/235584/e5c72fe6-09f4-43d4-8996-bbc49c59deb2)

Reply:
Alright. Harus diganti juga yang di #582. @mhaidarhanif